### ISSUE TITLE
[BUG] modules.list was not found

### ISSUE THREAD

**Issue by mdkumargce** [ElderDrivers/EdXposed#310]
What happened?**

Modules are not loading

Xposed Module List**

swiftkeyexi

Magisk Module List**

youtube vanced 

**EdXposed及Riru版本/Versions of EdXposed and Riru**

EdXposed: v0.4.5.1_beat(4463) Sandhook

Riru:v19

**相关Logcat/Logcat**
--------- beginning of head
EdXposed Log
Powered by Log Catcher
QQ chat group 855219808
--------- beginning of system info
Android version: 8.0.0
Android sdk: 26
Android build: OPR1.170623.032
Fingerprint: lge/h1_tmo_us/h1:8.0.0/OPR1.170623.032/182251913485f:user/release-keys
ROM build description: h1_tmo_us-user 8.0.0 OPR1.170623.032 182251913485f release-keys
EdXposed Version: v0.4.5.1_beta(4463) (Sandhook)
Architecture: arm64-v8a
Device: h1
Manufacture: LGE
Brand: lge
Product: h1
--------- beginning of main
--------- beginning of system
--------- beginning of crash
06-24 11:57:55.851  1939  1939 I EdXposed: onModuleLoaded: welcome to EdXposed!
06-24 11:57:55.851  1939  1939 I EdXposed: Start to install inline hooks
06-24 11:57:55.851  1939  1939 I EdXposed: Using api level 26
06-24 11:57:55.851  1939  1939 I EdXposed: Start to install Riru hook
06-24 11:57:55.864  1939  1939 I EdXposed: Riru hooks installed
06-24 11:57:55.879  1939  1939 I EdXposed: ART hooks installed
06-24 11:57:55.915  1939  1939 I EdXposed: system_property_get: dalvik.vm.dex2oat-filter -> quicken
06-24 11:57:55.915  1939  1939 I EdXposed: system_property_get: dalvik.vm.dex2oat-flags -> --inline-max-code-units=0
06-24 11:57:56.055  1939  1939 I EdXposed: using installer org.meowcat.edxposed.manager
06-24 11:57:56.055  1939  1939 I EdXposed: black/white list mode: false, using whitelist: false
06-24 11:57:56.055  1939  1939 I EdXposed: dynamic modules mode: false
06-24 11:57:56.056  1939  1939 I EdXposed: resources hook: true
06-24 11:57:56.056  1939  1939 I EdXposed: deopt boot image: false
06-24 11:57:56.082  1940  1940 I EdXposed: onModuleLoaded: welcome to EdXposed!
06-24 11:57:56.082  1940  1940 I EdXposed: Start to install inline hooks
06-24 11:57:56.082  1940  1940 I EdXposed: Using api level 26
06-24 11:57:56.082  1940  1940 I EdXposed: Start to install Riru hook
06-24 11:57:56.096  1940  1940 I EdXposed: Riru hooks installed
06-24 11:57:56.103  1940  1940 I EdXposed: ART hooks installed
06-24 11:57:56.171  1940  1940 I EdXposed: system_property_get: dalvik.vm.dex2oat-filter -> quicken
06-24 11:57:56.171  1940  1940 I EdXposed: system_property_get: dalvik.vm.dex2oat-flags -> --inline-max-code-units=0
06-24 11:57:56.322  1940  1940 I EdXposed: using installer org.meowcat.edxposed.manager
06-24 11:57:56.323  1940  1940 I EdXposed: black/white list mode: false, using whitelist: false
06-24 11:57:56.323  1940  1940 I EdXposed: dynamic modules mode: false
06-24 11:57:56.323  1940  1940 I EdXposed: resources hook: true
06-24 11:57:56.323  1940  1940 I EdXposed: deopt boot image: false
06-24 11:57:58.918  1939  1939 I EdXposed: EdxpVariant: 2
06-24 11:57:59.102  1939  1939 E EdXposed-Bridge: Cannot load any modules because /data/user_de/0/org.meowcat.edxposed.manager/conf/modules.list was not found
06-24 11:58:04.786  1940  1940 I EdXposed: EdxpVariant: 2
06-24 11:58:04.864  1940  1940 E EdXposed-Bridge: Cannot load any modules because /data/user_de/0/org.meowcat.edxposed.manager/conf/modules.list was not found
06-24 12:02:38.114 13982 13982 I EdXposed: onModuleLoaded: welcome to EdXposed!
06-24 12:02:38.114 13982 13982 I EdXposed: Start to install inline hooks
06-24 12:02:38.114 13982 13982 I EdXposed: Using api level 26
06-24 12:02:38.114 13982 13982 I EdXposed: Start to install Riru hook
06-24 12:02:38.127 13982 13982 I EdXposed: Riru hooks installed
06-24 12:02:38.140 13982 13982 I EdXposed: ART hooks installed
06-24 12:02:38.183 13982 13982 I EdXposed: system_property_get: dalvik.vm.dex2oat-filter -> quicken
06-24 12:02:38.184 13982 13982 I EdXposed: system_property_get: dalvik.vm.dex2oat-flags -> --inline-max-code-units=0
06-24 12:02:38.327 13982 13982 I EdXposed: using installer org.meowcat.edxposed.manager
06-24 12:02:38.327 13982 13982 I EdXposed: black/white list mode: false, using whitelist: false
06-24 12:02:38.327 13982 13982 I EdXposed: dynamic modules mode: false
06-24 12:02:38.327 13982 13982 I EdXposed: resources hook: true
06-24 12:02:38.327 13982 13982 I EdXposed: deopt boot image: false
06-24 12:02:40.490 13982 13982 I EdXposed: EdxpVariant: 2
06-24 12:02:40.652 13982 13982 E EdXposed-Bridge: Cannot load any modules because /data/user_de/0/org.meowcat.edxposed.manager/conf/modules.list was not found

有助于定位问题，必须使用我们的log抓取模块抓取/It can help us to locate issue, must use our logcat module

**Comment by mdkumargce** [comment]
EdXposed manager was not able to find modules.list even when whitelist is disabled.

**Comment by NewBugger** [comment]
i've also met this issue, you should completely uninstall Edxposed and its manager/installer, and reinstall it all.

**Comment by eebssk1** [comment]
Also check  your dmesg to see if reading  module.list  is blocked by SELinux

**Comment by mdkumargce** [comment]
it appears that SELinux was the problem. setting it to permissive made exposed manager load all the modules. is there any other way to make this work by still keeping SELinux Enforced?

**Comment by eebssk1** [comment]
> 
> 
> it appears that SELinux was the problem. setting it to permissive made exposed manager load all the modules. is there any other way to make this work by still keeping SELinux Enforced?

may related to #314

**Comment by giga** [comment]
How can I change SELinux to permissive? I tried SELinux Switch app but it does not solve theh issue.

**Comment by giga** [comment]
Never mind, this was fixed for me by using custom Magisk from https://github.com/solohsu/magisk_files

**Comment by heatl0rd** [comment]
I keep getting `Cannot load any modules because /data/user_de/0/org.meowcat.edxposed.manager/conf/modules.list was not found` 

So i tried the custom Magisk from https://github.com/solohsu/magisk_files by using the custom URL and clicking 'install' next to Magisk (not magisk manager) and it installs 20.1 (20101)

But then i get softbrick. Phone is stuck at pixel logo.

My stats are
`--------- beginning of head
EdXposed Log
Powered by Log Catcher
QQ chat group 855219808
--------- beginning of system info
Android version: 10
Android sdk: 29
Android build: QQ1B.200205.002
Fingerprint: google/coral/coral:10/QQ1B.200205.002/6084387:user/release-keys
ROM build description: coral-user 10 QQ1B.200205.002 6084387 release-keys
EdXposed Version: v0.4.6.0_beta(4471) (Sandhook)
Architecture: arm64-v8a
Device: coral
Manufacture: Google
Brand: google
Product: coral
--------- beginning of main
--------- beginning of system
02-27 19:30:53.426   876   876 I EdXposed: onModuleLoaded: welcome to EdXposed!
02-27 19:30:53.426   876   876 I EdXposed: Start to install inline hooks
02-27 19:30:53.426   876   876 I EdXposed: Using api level 29
02-27 19:30:53.426   876   876 I EdXposed: Start to install Riru hook
02-27 19:30:53.435   876   876 I EdXposed: Riru hooks installed
02-27 19:30:53.536   876   876 I EdXposed: ART hooks installed
02-27 19:30:53.536   876   876 I EdXposed: system_property_get: dalvik.vm.dex2oat-filter -> quicken
02-27 19:30:53.536   876   876 I EdXposed: system_property_get: dalvik.vm.dex2oat-flags -> --inline-max-code-units=0
02-27 19:30:53.986   877   877 I EdXposed: onModuleLoaded: welcome to EdXposed!
02-27 19:30:53.987   877   877 I EdXposed: Start to install inline hooks
02-27 19:30:53.987   877   877 I EdXposed: Using api level 29
02-27 19:30:53.987   877   877 I EdXposed: Start to install Riru hook
02-27 19:30:54.001   877   877 I EdXposed: Riru hooks installed
02-27 19:30:54.019   876   876 I EdXposed: using installer org.meowcat.edxposed.manager
02-27 19:30:54.019   876   876 I EdXposed: black/white list mode: false, using whitelist: false
02-27 19:30:54.019   876   876 I EdXposed: dynamic modules mode: false
02-27 19:30:54.019   876   876 I EdXposed: resources hook: true
02-27 19:30:54.019   876   876 I EdXposed: deopt boot image: false
02-27 19:30:54.106   877   877 I EdXposed: ART hooks installed
02-27 19:30:54.106   877   877 I EdXposed: system_property_get: dalvik.vm.dex2oat-filter -> quicken
02-27 19:30:54.106   877   877 I EdXposed: system_property_get: dalvik.vm.dex2oat-flags -> --inline-max-code-units=0
02-27 19:30:54.272   877   877 I EdXposed: using installer org.meowcat.edxposed.manager
02-27 19:30:54.272   877   877 I EdXposed: black/white list mode: false, using whitelist: false
02-27 19:30:54.272   877   877 I EdXposed: dynamic modules mode: false
02-27 19:30:54.272   877   877 I EdXposed: resources hook: true
02-27 19:30:54.272   877   877 I EdXposed: deopt boot image: false
02-27 19:30:55.403   876   876 E EdXposed-Bridge: Cannot load any modules because /data/user_de/0/org.meowcat.edxposed.manager/conf/modules.list was not found
02-27 19:31:07.812   877   877 E EdXposed-Bridge: Cannot load any modules because /data/user_de/0/org.meowcat.edxposed.manager/conf/modules.list was not found
--------- beginning of crash
`

**Comment by eebssk1** [comment]
Custom magisk is deprecated.Try newest canary magisk and canny EdXposed.

**Comment by Justsnoopy30** [comment]
Still an issue...

**Comment by Justsnoopy30** [comment]
@MlgmXyysd

**Comment by MlgmXyysd** [comment]
> @MlgmXyysd

So you should open new issue to provide your own infomation
